package.json

```diff
@@ -34,6 +34,7 @@
     "gulp": "^4.0.0",
     "history": "^4.7.2",
     "jest": "^23.1.0",
+    "js-cookie": "^2.2.0",
```

Comment:
This should handle any cross-browser compatibility weirdness, on top of allowing us to delete custom cookie-handling code.